packages/firebase_storage/test/firebase_storage_test.dart

```diff
@@ -53,6 +53,46 @@ void main() {
       });
     });
 
+    group('getDownloadUrl', () {
+      const MethodChannel channel = const MethodChannel(
```

Comment:
This plugin should match the other plugins in terms of the method channel name: `'plugins.flutter.io/firebase_storage'`. This is can be done in a separate CL, I filed flutter/flutter#16171

Reply:
Noted.